kits/defect-triage-agent/prompts/defect-triage_instructor-llmnode-359_user_1.md

```
@@ -0,0 +1 @@
+Defect observation:{{triggerNode_1.output.defectDescription}}
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)